gc/gc-base/src/test/java/org/projectnessie/gc/repository/TestNessieRepositoryConnector.java

```diff
@@ -39,7 +40,6 @@
 import org.projectnessie.client.ext.NessieClientFactory;
 import org.projectnessie.error.NessieConflictException;
 import org.projectnessie.error.NessieNotFoundException;
-import org.projectnessie.jaxrs.ext.NessieJaxRsExtension;
```

Comment:
nit: same here: why not keep the import?

Reply:
to use a simple class name on line 70... same in `TestIdentifyLiveContents.java`

Reply:
It's in the static-import - that's why the method name's that verbose.

Reply:
I mean the `sever` variable type.

Reply:
oh - no clue why IJ changed that